docs/src/main/sphinx/functions/list-by-topic.rst

```diff
@@ -0,0 +1,562 @@
+=========================================
+List of functions and operators per topic
+=========================================
+
+Aggregate
+---------
+
+Details in :doc:`aggregate`.
+
+- :func:`arbitrary`
+- :func:`approx_distinct`
+- :func:`approx_most_frequent`
+- :func:`approx_percentile`
+- :func:`approx_set`
+- :func:`array_agg`
+- :func:`avg`
+- :func:`bitwise_and_agg`
+- :func:`bitwise_or_agg`
+- :func:`bool_and`
+- :func:`bool_or`
+- :func:`checksum`
+- :func:`corr`
+- :func:`count`
+- :func:`count_if`
+- :func:`covar_pop`
+- :func:`covar_samp`
+- :func:`every`
+- :func:`geometric_mean`
+- :func:`histogram`
+- :func:`kurtosis`
+- :func:`map_agg`
+- :func:`map_union`
+- :func:`max`
+- :func:`max_by`
+- :func:`merge`
+- :func:`min`
+- :func:`min_by`
+- :func:`multimap_agg`
+- :func:`numeric_histogram`
+- :func:`qdigest_agg`
+- :func:`regr_intercept`
+- :func:`regr_slope`
+- :func:`skewness`
+- :func:`sum`
+- :func:`stddev`
+- :func:`stddev_pop`
+- :func:`stddev_samp`
+- :func:`tdigest`
+- :func:`variance`
+- :func:`var_pop`
+- :func:`var_samp`
+
+Array
+-----
+
+Details in :doc:`array`.
+
+Concatenation operator: ||
```

Comment:
Let's remove all the operator references here. The extra section headers should also be removed.